posthog/models/event_definition.py

```diff
@@ -20,8 +20,12 @@ class EventDefinition(UUIDModel):
     class Meta:
         unique_together = ("team", "name")
         indexes = [
+            # To speed up DB-based fuzzy searching
             GinIndex(name="index_event_definition_name", fields=["name"], opclasses=["gin_trgm_ops"]),
-        ]  # To speed up DB-based fuzzy searching
+            # Improve ordering performance
+            models.Index(fields=["team_id", "-volume_30_day"]),
```

Comment:
I'm curious how indexing works with the `-`? Do you know any good literature on this?

Reply:
It should just add `desc` in the index as well, and should actually make no difference.

Reply:
Postgres-level does that just mean better performance because we would order desc too? What's the impact of not having the `desc` in there?

Reply:
Sorry, forgot a link: https://www.postgresql.org/docs/14/indexes-ordering.html

I copied this from a different model in the system, and left the `-` in place since we mostly do order DESC here. If I read the doc correctly, there shouldn't be any real difference between the two. This will be truly useful if we'd sort by multiple columns. We don't here, and postgres can read indexes backwards, so it's probably better to skip the `-` here for clarity.

At least I should run a few tests on it to make sure it works well, but now it's gone, so :).